src/Tribe/Main.php

```diff
@@ -40,12 +35,53 @@ public static function instance() {
 	 */
 	protected function __construct() {
 
+		$this->init();
 
 	}
 
 	public function init() {
 
+		// Setup bootstrap on earliest hook available to the extension.
+		add_action( 'tribe_plugins_loaded', [ $this, 'bootstrap' ], 11 );
+
+		$this->get_settings();
+	}
+
+	/**
+	 * Bootstrap Plugin
+	 *
+	 * @since TBD
+	 *
+	 */
+	public function bootstrap() {
+
+		// Intialize the Service Provider for Event Tickets HubSpot Integration.
+		tribe_register_provider( Service_Provider::class );
+
+	}
+
+	/**
+	 * Get Settings instance.
+	 *
+	 * @return Settings
+	 */
+	private function get_settings() {
+		if ( empty( $this->settings ) ) {
+			$this->settings = new Settings();
+		}
+
+		return $this->settings;
+	}
+
+	/**
+	 * Get all of this extension's options.
+	 *
+	 * @return array
+	 */
+	public function get_all_options() {
+		$settings = $this->get_settings();
```

Comment:
```suggestion
		$settings = tribe( 'tickets.hubspot.admin.settings' );
```

Reply:
Changed to tribe( 'tickets.hubspot.admin.settings' ).